<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Contribution</title>
    <link rel="stylesheet" href="css/bootstrap.min.css">
    <script src="js/jquery-3.5.1.min.js"></script>
    <script src="js/bootstrap.min.js"></script>
    <link rel="stylesheet" href="css/general.css">
    <script src="js/general.js"></script>
    <link rel="stylesheet" href="css/sidebar.css">
    <link rel="stylesheet" href="css/article.css">
    <script src="js/sidebar.js"></script>
</head>
<body>
<nav id="nav-bar" class="navbar navbar-expand-lg navbar-dark sticky-top">
    <a class="navbar-brand" href="https://2020.igem.org/Team:DUT_China">
        <img src="https://2020.igem.org/wiki/images/d/d4/T--DUT_China--team_logo.svg" alt="DUT_China">
    </a>
    <button class="navbar-toggler" data-toggle="collapse" data-target="#nav1">
        <span class="navbar-toggler-icon"></span>
    </button>
    <div class="collapse navbar-collapse" id="nav1">
        <ul class="navbar-nav ml-auto">
            <li class="nav-item">
                <a class="nav-link" href="https://2020.igem.org/Team:DUT_China">
                    <img class="nav-logo" src="https://2020.igem.org/wiki/images/4/4a/T--DUT_China--n_home1.png" alt="">
                    <span>Home</span>
                </a>
            </li>
            <li class="nav-item dropdown">
                <a class="nav-link dropdown-toggle" data-toggle="dropdown">
                    <img class="nav-logo" src="https://2020.igem.org/wiki/images/3/3f/T--DUT_China--n_project1.png"
                         alt="">
                    <span>Project</span>
                </a>
                <div class="dropdown-menu">
                    <a class="dropdown-item" href="https://2020.igem.org/Team:DUT_China/Description">Description</a>
                    <a class="dropdown-item" href="https://2020.igem.org/Team:DUT_China/Engineering">Engineering
                        Success</a>
                    <a class="dropdown-item" href="https://2020.igem.org/Team:DUT_China/Protocols">Protocols</a>
                    <a class="dropdown-item" href="https://2020.igem.org/Team:DUT_China/Contribution">Contribution</a>
                </div>
            </li>
            <li class="nav-item dropdown">
                <a class="nav-link dropdown-toggle" data-toggle="dropdown">
                    <img class="nav-logo" src="https://2020.igem.org/wiki/images/4/4d/T--DUT_China--n_model.png" alt="">
                    <span>Model</span>
                </a>
                <div class="dropdown-menu">
                    <a class="dropdown-item" href="https://2020.igem.org/Team:DUT_China/Model">Electroporation</a>
                </div>
            </li>
            <li class="nav-item dropdown">
                <a class="nav-link dropdown-toggle" data-toggle="dropdown">
                    <img class="nav-logo" src="https://2020.igem.org/wiki/images/2/23/T--DUT_China--n_hp1.png" alt="">
                    <span>Human Practices</span>
                </a>
                <div class="dropdown-menu">
                    <a class="dropdown-item" href="https://2020.igem.org/Team:DUT_China/Human_Practices">Integrated
                        Human Practices</a>
                    <a class="dropdown-item" href="https://2020.igem.org/Team:DUT_China/Education">Education</a>
                </div>
            </li>
            <li class="nav-item dropdown">
                <a class="nav-link dropdown-toggle" data-toggle="dropdown">
                    <img class="nav-logo"
                         src="https://2020.igem.org/wiki/images/f/f6/T--DUT_China--n_implementation1.png" alt="">
                    <span>Implementation</span>
                </a>
                <div class="dropdown-menu">
                    <a class="dropdown-item" href="https://2020.igem.org/Team:DUT_China/Implementation">Business
                        Proposals</a>
                    <a class="dropdown-item" href="https://2020.igem.org/Team:DUT_China/Legislative">Legislative
                        Proposals</a>
                </div>
            </li>
            <li class="nav-item dropdown">
                <a class="nav-link dropdown-toggle" data-toggle="dropdown">
                    <img class="nav-logo" src="https://2020.igem.org/wiki/images/2/29/T--DUT_China--n_team1.png" alt="">
                    <span>Team</span>
                </a>
                <div class="dropdown-menu">
                    <a class="dropdown-item" href="https://2020.igem.org/Team:DUT_China/Team">Members</a>
                    <a class="dropdown-item" href="https://2020.igem.org/Team:DUT_China/Attributions">Attributions</a>
                    <a class="dropdown-item"
                       href="https://2020.igem.org/Team:DUT_China/Collaborations">Collaborations</a>
                </div>
            </li>
            <li class="nav-item">
                <a class="nav-link" href="https://2020.igem.org/Team:DUT_China/Safety">
                    <img class="nav-logo" src="https://2020.igem.org/wiki/images/0/04/T--DUT_China--n_safety1.png"
                         alt="">
                    <span>Safety</span>
                </a>
            </li>
            <li class="nav-item dropdown">
                <a class="nav-link dropdown-toggle" data-toggle="dropdown">
                    <img class="nav-logo" src="https://2020.igem.org/wiki/images/b/bb/T--DUT_China--n_awards1.png"
                         alt="">
                    <span>Awards</span>
                </a>
                <div class="dropdown-menu">
                    <a class="dropdown-item" href="https://2020.igem.org/Team:DUT_China/Human_Practices">Integrated
                        Human Practices</a>
                    <a class="dropdown-item" href="https://2020.igem.org/Team:DUT_China/Model">Model</a>
                    <a class="dropdown-item" href="https://2020.igem.org/Team:DUT_China/Education">Education</a>
                </div>
            </li>
        </ul>
    </div>
</nav>
<div id="top-scroll">
    <svg class="icon" style="width: 100px; height: 100px;vertical-align: middle;fill: #0073bb;overflow: hidden;"
         viewBox="0 0 1024 1024" version="1.1" xmlns="http://www.w3.org/2000/svg" p-id="2424">
        <path d="M4.621 514.463h98.188v327.301h98.188v-327.301h98.188v-65.449h-294.563zM528.209 154.51l-163.446 196.347 327.273-0.219zM331.907 841.765h327.286v-392.75h-327.286v392.75zM430.095 514.463h130.912v261.823h-130.912v-261.823zM691.933 449.015v392.75h98.188v-163.651h229.1v-229.1h-327.286zM921.033 612.652h-130.912v-98.188h130.912v98.188z"
              p-id="2425"></path>
    </svg>
</div>
<div id="sidebar">
    <table>
        <tr>
            <td>
                <canvas width="40" height="40" class="canvas"></canvas>
            </td>
            <td class="side-item">1 Our First Contribution</td>
        </tr>
        <tr>
            <td>
                <canvas width="40" height="40" class="canvas"></canvas>
            </td>
            <td class="side-item">2 Our Second Contribution</td>
        </tr>
        <tr>
            <td>
                <canvas width="40" height="40" class="canvas"></canvas>
            </td>
            <td class="side-item">3 Our Third Contribution</td>
        </tr>
        <tr>
            <td>
                <canvas width="40" height="40" class="canvas"></canvas>
            </td>
            <td class="side-item">4 Background Information</td>
        </tr>
        <tr>
            <td>
                <canvas width="40" height="40" class="canvas"></canvas>
            </td>
            <td class="side-item">5 Appendix</td>
        </tr>
        <tr>
            <td>
                <canvas width="40" height="40" class="canvas"></canvas>
            </td>
            <td class="side-item">6 References</td>
        </tr>
    </table>
</div>
<div id="title" class="col-12">
    <div>Contribution</div>
</div>
<section id="main-text">
    <p>
        Our contributions are to sort out the genome information of T4 phage, replenish the gene information of phage T4 genome according to the latest results (More details are in the appendix) and made a new genome annotation map, which provides accessible and reliable data for later researchers to reboot engineered phages.
    </p>
    <article class="article">
        <h1>1 Our First Contribution</h1>
        <div class="too-long">
            <img src="https://2020.igem.org/wiki/images/3/35/T--DUT_China--p_con_0.svg" alt="">
        </div>
        <p class="caption"><b>Figure 1.</b> Interactive genome annotation.</p>
        <p>
            An interactive figure is shown above in which you can easily get access to the information of T4 phage genes in hover position. If the figure above doesn't work, click the link to view the interactive version: <a href="https://2020.igem.org/Team:DUT_China/T4_phage">https://2020.igem.org/Team:DUT_China/T4_phage</a>
        </p>
        <p>
            The information are essential gene names, position, length, weight and product in this figure. We use colors of blue, purple, orange and yellow to represent the part responsible for encoding tail structure, head structure, energy control system and central dogma system, respectively.
        </p>
        <p>
            Through genome annotation above, our phage reboot technology can be applied with convenience. By this, future researchers can easily segment the phage genome according to whether it is conservative or not, and add the variable and non-variable parts into different plasmids. Only the plasmids with variable parts (such as knockout gene) are edited, and then researchers can copy these plasmids through yeast cells and then transfer them into <i>E. coli</i>. And in the next step, phages reboot. Therefore, original complex phage genome editing (through gene recombination or non-cellular system) can be innovatively transformed into simple plasmid editing.
        </p>
    </article>
    <article class="article">
        <h1>2 Our Second Contribution</h1>
        <p>
            The second contribution of our team is the table of T4 phage genome annotations below (<b>Table 1</b>), which provides latter researchers a more convenient platform to identify genes that can be edited and not. There are several hundred CDS in T4-like phage's genomes, in which some of them can be modified or changed to attain engineering and scientific aims, while others are essential and immutable. And in order to make it clearer, here we made a list with different colors to reveal different types of conservative parts of genes in <b>Table 1</b>. Relevant information is listed in <b>Table 2</b>.
        </p>
        <p class="caption"><b>Table 1.</b> Essential genes.</p>
        <img src="https://2020.igem.org/wiki/images/9/9c/T--DUT_China--p_con_1_1.png" alt="" class="col-12">
        <p class="caption"><b>Table 2.</b> The legend of different types of essential genes.</p>
        <img src="https://2020.igem.org/wiki/images/5/58/T--DUT_China--p_con_2_1.png" alt="" class="col-sm-12 col-md-6 mx-auto">
    </article>
    <article class="article">
        <h1>3 Our Third Contribution</h1>
        <p>
            Our third contribution is to replenish relevant information according to the latest results (More details are in the appendix) and made sequences alignment (<b>Fig.2</b>) to compare 4 kinds of <i>E. coli</i> phages, <i>T4, T6, slur03, KIT03</i>.
        </p>
        <img src="https://2020.igem.org/wiki/images/f/f4/T--DUT_China--p_con_3.svg" alt="" class="col-12">
        <p class="caption"><b>Figure 2.</b> Bioinformatic analysis of phages DNA sequences alignment.</p>
        <p>
            Our sequences alignment result has revealed relationships between T4, T6, slur03, KIT03. Five kinds arrows with different colors reveal different essential functions, which gives us an evidence to ensure our judgement.
        </p>
        <p>And in sequences alignment above:</p>
        <p>1.Only sequence similarities more than 100 base pairs are listed here.</p>
        <p>2.The possible functions of <i>T6, KIT03</i> and <i>slur03</i> phages sequence fragments are inferred from T4.</p>
        <p>3.Expectation value is 0.001. </p>
        <p>4.The scoring matrix used here is BLOSUM62. </p>
        <p>5.Here we use Easyfig software to draw the graph above<sup>[1]</sup>.</p>
    </article>
    <article class="article">
        <h1>4 Background Information</h1>
        <p>
            T4 shows nearly four times the gene density predicted for herpesviruses and yeast and twice that for <i>E. coli</i><sup>[2-4]</sup>. The high gene density reflects both the small size of many T4 genes and the fact that there are very few noncoding regions (about 9 kb, 5.3% of the genome). Furthermore, regulatory regions are compact, occasionally overlapping coding regions. In many cases, the termination codon of one gene overlaps the start codon of the next gene (see “Translation and posttranscriptional control” below). In addition, T4 has several groups of nested genes as mentioned above. Clearly, computational and bioinformatic tools do not yet identify all the genes and complex coding arrangements in a genome perceived by many to be “simple,” like that of T4. Only 62 of the T4 genes are “essential” under standard laboratory conditions (rich medium, aeration, 30 to 37°C)<sup>[5]</sup>; mutants altered in a few other genes produce very small plaques under standard conditions. Many of these key genes are much larger than the average T4 gene; together, they occupy almost half of the genome. They include genes that encode proteins of the replisome and of the nucleotide-precursor complex, several transcriptional regulatory factors, and most of the structural and assembly proteins of the phage particle.
        </p>
    </article>
    <article class="article">
        <h1>5 Appendix</h1>
        <h2>1. SegD</h2>
        <p>A novel phage T4 endonuclease, SegD, recognizes an extended, 16 bp long, site, cleaves it asymmetrically to form 3′-protruding ends and digests both unmodified DNA and modified T-even phage DNA with similar efficiencies.
            Homing endonucleases are a group of site-specific endonucleases that initiate homing, a nonreciprocal transfer of its own gene into a new allele lacking this gene. Endonuclease SegD is homologous to the GIY-YIG family of homing endonucleases. However, SegD was not able to initiate homing of its own gene<sup>[6]</sup>.</p>
        <h2>2. RNase E</h2>
        <p>Early gene expression in bacteriophage T4 is controlled primarily by the unique early promoters, while T4-encoded RegB endoribonuclease promotes degradation of many early messages contributing to the rapid shift of gene expression from the early to middle stages.
            The initial cleavage by RegB of phages TuIa and RB69 enables degradation of early phage mRNAs by the major <i>Escherichia coli</i> endoribonuclease, RNase E<sup>[7]</sup>.</p>
        <h2>3. g23(capsid gene)</h2>
        <p>
            Genomic information is preserved among viral subsets and can be used for phylogenetic classification of viruses and for evaluation of viral diversity in the environment. The capsid gene of T4-type bacteriophages, g23, is the most widely applied gene for evaluating the diversity of the T4-type bacteriophage family<sup>[8-11]</sup>.
        </p>
        <h2>4. gp4</h2>
        <p>
            gp4 (phage T4 gene product 4) plays an essential role in the head-tail joining step of T4-like phages. Amber mutations in gene 4 prevent heads and tails from joining efficiently and 50% of the heads produced lack DNA<sup>[12]</sup>.
        </p>
        <h2>5. UvsY(recombination mediator protein)</h2>
        <p>
            Bacteriophage T4 UvsY is a recombination mediator protein that promotes assembly of the UvsX-ssDNA presynaptic filament. UvsY helps UvsX to displace T4 gene 32 protein (gp32) from ssDNA, a reaction necessary for proper formation of the presynaptic filament.
            UvsY interacts strongly with stretched DNA in the absence of other proteins. In the presence of gp32 and *I, UvsY is capable of strongly destabilizing gp32-DNA complexes in order to facilitate ssDNA wrapping, which in turn prepares the ssDNA for presynaptic filament assembly in the presence of UvsX. Thus, UvsY mediates UvsX binding to ssDNA by converting rigid gp32-DNA filaments into a structure that can be strongly bound by UvsX<sup>[13]</sup>.
        </p>
        <h2>6. The baseplate protein gene product</h2>
        <p>
            The proteins with known atomic structures: gp 5, gp8, gp9, gp11, gp12, and gp27.
        </p>
        <p>
            The proteins whose structures had not yet been established: gp6, gp7, gp10, gp25, gp29, gp48, gp53, and gp54.
        </p>
        <p>
            During the attachment of the phage to the host, the baseplate changes its conformation from being a hexagonally shaped dome into a planar six-pointed star structure. This conformational change triggers contraction of the sheath, which results in penetration of the cell envelope by the tail tube, allowing the phage to transfer its DNA into the host cell<sup>[14]</sup>.
        </p>
        <h2>7. holin protein (T); antiholin (RI)</h2>
        <p>
            In T4 phages, the length and fecundity of the infection cycle are determined by small membrane proteins called holins and their specific regulators, the antiholins.
            Holins initiate lysis by permeabilizing the cytoplasmic membrane at a genetically programmed time, setting in motion a lysis pathway that results in release of the viral progeny.
            Free periplasmic RI, which would bind to the periplasmic domain of T and prevent it from making holes. The structure of free RI shows a unique fold, capable of undergoing a large conformational change and domain swapping, which facilitates the release of the holin from the complex upon lysis triggering<sup>[15]</sup>.
        </p>
        <h2>8. gp24(vertex protein)</h2>
        <p>
            The capsid of bacteriophage T4 is composed of the major capsid protein gp23, and a minor capsid protein gp24. Two glutamate to alanine mutations (E89A,E90A) can improve the stability, crystallizability and crystal quality of gp24 without affecting the overall folding.
            Rational modification of the protein surface to achieve crystallization appears promising for improving crystallization behavior and crystal diffracting qualities. Surface mutation proved to be a better method than reductive methylation for improving diffraction quality of the gp24 crystals<sup>[16]</sup>.
        </p>
    </article>
    <article class="article">
        <h1>6 References</h1>
        <p>[1]Mitchell, J., et al., Easyfig: a genome comparison visualizer. <i>Bioinformatics</i> (Oxford, England), 2011.</p>
        <p>[2]Blattner, F. R., Plunkett, G., Bloch, C. A., Perna, N. T., Burland, V., Riley, M., ... & Gregor, J. (1997). The complete genome sequence of Escherichia coli K-12. <i>Science, 277</i>(5331), 1453-1462.</p>
        <p>[3]Koonin, E. V., Aravind, L., & Kondrashov, A. S. (2000). The impact of comparative genomics on our understanding of evolution. <i>Cell, 101</i>(6), 573-576.</p>
        <p>[4]Koonin, E. V., Bork, P., & Sander, C. (1994). Yeast chromosome III: new gene functions. <i>The EMBO Journal, 13</i>(3), 493-503.</p>
        <p>[5]Miller, E. S., Kutter, E., Mosig, G., Arisaka, F., Kunisawa, T., & Rüger, W. (2003). Bacteriophage T4 genome. <i>Microbiology and molecular biology reviews, 67</i>(1), 86-156.</p>
        <p>[6]Sokolov, A. S., Latypov, O. R., Kolosov, P. M., Shlyapnikov, M. G., Bezlepkina, T. A., Kholod, N. S., ... & Granovsky, I. E. (2018). Phage T4 endonuclease SegD that is similar to group I intron endonucleases does not initiate homing of its own gene. <i>Virology, 515</i>, 215-222.</p>
        <p>[7]Truncaite, L., Zajančkauskaite, A., Arlauskas, A., & Nivinskas, R. (2006). Transcription and RNA processing during expression of genes preceding DNA ligase gene 30 in T4-related bacteriophages. <i>Virology, 344</i>(2), 378-390.</p>
        <p>[8]Fujii, T., Nakayama, N., Nishida, M., Sekiya, H., Kato, N., Asakawa, S., & Kimura, M. (2008). Novel capsid genes (g23) of T4-type bacteriophages in a Japanese paddy field. <i>Soil Biology and Biochemistry, 40</i>(5), 1049-1058.</p>
        <p>[9]Filée, J., Tétart, F., Suttle, C. A., & Krisch, H. M. (2005). Marine T4-type bacteriophages, a ubiquitous component of the dark matter of the biosphere. <i>Proceedings of the National Academy of Sciences, 102</i>(35), 12471-12476.</p>
        <p>[10]Liu, J., Wang, G., Zheng, C., Yuan, X., Jin, J., & Liu, X. (2011). Specific assemblages of major capsid genes (g23) of T4-type bacteriophages isolated from upland black soils in Northeast China. <i>Soil Biology and Biochemistry, 43</i>(9), 1980-1984.</p>
        <p>[11]Jia, Z., Ishihara, R., Nakajima, Y., Asakawa, S., & Kimura, M. (2007). Molecular characterization of T4‐type bacteriophages in a rice field. <i>Environmental Microbiology, 9</i>(4), 1091-1096.</p>
        <p>[12]Benler, S., Hung, S. H., Vander Griend, J. A., Peters, G. A., Rohwer, F., & Segall, A. M. (2020). Gp4 is a nuclease required for morphogenesis of T4-like bacteriophages. <i>Virology, 543</i>, 7-12.</p>
        <p>[13]Pant, K., Shokri, L., Karpel, R. L., Morrical, S. W., & Williams, M. C. (2008). Modulation of T4 gene 32 protein DNA binding activity by the recombination mediator protein UvsY. <i>Journal of molecular biology, 380</i>(5), 799-811.</p>
        <p>[14]Leiman, P. G., Shneider, M. M., Mesyanzhinov, V. V., & Rossmann, M. G. (2006). Evolution of bacteriophage tails: Structure of T4 gene product 10. <i>Journal of molecular biology, 358</i>(3), 912-921.</p>
        <p>[15]Krieger, I. V., Kuznetsov, V., Chang, J. Y., Zhang, J., Moussa, S. H., Young, R. F., & Sacchettini, J. C. (2020). The Structural Basis of T4 Phage Lysis Control: DNA as the Signal for Lysis Inhibition. <i>Journal of molecular biology, 432</i>(16), 4623-4636.</p>
        <p>[16]Boeshans, K. M., Liu, F., Peng, G., Idler, W., Jang, S. I., Marekov, L., ... & Ahvazi, B. (2006). Purification, crystallization and preliminary X-ray diffraction analysis of the phage T4 vertex protein gp24 and its mutant forms. <i>Protein expression and purification, 49</i>(2), 235-243.</p>
    </article>
    <p class="text-right">
        <a href="https://2020.igem.org/Team:DUT_China/Model">Model - Electroportation&#10148;</a>
    </p>
</section>
<footer id="footer">
    <img class="flower flower1" src="https://2020.igem.org/wiki/images/0/05/T--DUT_China--flower1.png" alt="">
    <img class="flower flower2" src="https://2020.igem.org/wiki/images/0/0b/T--DUT_China--flower2.png" alt="">
    <p class="copyright">© 2020 Dalian University of Technology iGEM Team</p>
    <div class="row">
        <div class="col-lg-4 footer-left">
            <h2><span>DUT</span> iGEM Players</h2>
            <img src="https://2020.igem.org/wiki/images/d/d7/T--DUT_China--school_badge1.svg" alt="School Logo">
            <p class="plain-text text-left">
                <b>Address</b>: No.2 Linggong Road, Ganjingzi District, Dalian City, Liaoning Province, P.R.C.,
                116024<br>
                <b>Official Website</b>: <a href="http://en.dlut.edu.cn/" target="_blank">http://en.dlut.edu.cn/</a><br>
                <b>Team Email</b>: <a href="mailto:igem_dlut@163.com" title="Click to contact us">igem_dlut@163.com</a>
            </p>
        </div>

        <div class="col-lg-4 footer-center">
            <div>
                <h4>About Dalian University of Technology</h4>
                <p class="plain-text school-intro text-left">
                    Dalian University of Technology (DUT) is a national key university administrated directly under the
                    Ministry of Education of China, which is also sponsored by Project 211 and Project 985. DUT keeps
                    fostering elites, promoting science and technology, inheriting excellent cultures and guiding the
                    social climate as its mission, adheres to DUT spirit of “Unity and Progress, Truth and Innovation”,
                    dedicates itself to the creation, discovery, impart, conservation and application of knowledge, and
                    endeavors to undertake social responsibilities to serve the country and the world.
                </p>
            </div>
        </div>

        <div class="col-lg-4 footer-right">
            <div>
                <h4>Follow us</h4>
                <p class="plain-text">
                    <a target="_blank" href="https://mp.weixin.qq.com/s/gDbIeINj-mwxmwDkOSyFQg">
                        WeChat Official Account
                    </a>
                    <br>
                    <a target="_blank" href="https://space.bilibili.com/494554805/">
                        Bilibili Account
                    </a>
                </p>
            </div>
            <br>
            <div>
                <h4>Sponsors</h4>
                <img src="https://2020.igem.org/wiki/images/2/29/T--DUT_China--sponsor_snapgene.png" alt="SnapGene">
                <br>
                <img src="https://2020.igem.org/wiki/images/e/e3/T--DUT_China--biology1.svg" alt="School of Bioengineering">
            </div>
        </div>
    </div>
</footer>
<script>
    //About the sidebar & top-scroll--linked to sidebar.js & general.js
    window.onload = function () {
        let navHeight = document.getElementById('nav-bar').offsetHeight;
        for (let i = 0; i < itemArray.length; i++) {
            itemArray[i].addEventListener('click', function () {
                scroll({
                    top: articleArray[i].offsetTop - navHeight + 5,
                    //To prevent the overshadow of the navigation bar
                    left: 0,
                    behavior: 'smooth'
                });
            });
            //Click the items to scroll
            pageView(canvasArray[i], itemArray[i], articleArray[i]);//Update
        }
        setTopScroll();
    };

    window.onresize = window.onscroll = function () {
        for (let i = 0; i < canvasArray.length; i++) {
            pageView(canvasArray[i], itemArray[i], articleArray[i]);//Update
        }
        let textT = document.getElementById("main-text").offsetTop;
        let currentH = document.documentElement.scrollTop || document.body.scrollTop;
        let sidebar = document.getElementById("sidebar");
        let topScroll = document.getElementById("top-scroll");
        if (currentH > textT - 130) {
            sidebar.setAttribute("class", "sidebar-active");
            topScroll.setAttribute("class", "top-scroll-active");
        } else {
            sidebar.setAttribute("class", "");
            topScroll.setAttribute("class", "");
        }
    };
</script>
</body>
</html>