<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Engineering Success</title>
    <link rel="stylesheet" href="css/bootstrap.min.css">
    <script src="js/jquery-3.5.1.min.js"></script>
    <script src="js/bootstrap.min.js"></script>
    <link rel="stylesheet" href="css/general.css">
    <script src="js/general.js"></script>
    <link rel="stylesheet" href="css/sidebar.css">
    <link rel="stylesheet" href="css/article.css">
    <script src="js/sidebar.js"></script>
</head>
<body>
<nav id="nav-bar" class="navbar navbar-expand-lg navbar-dark sticky-top">
    <a class="navbar-brand" href="https://2020.igem.org/Team:DUT_China">
        <img src="https://2020.igem.org/wiki/images/d/d4/T--DUT_China--team_logo.svg" alt="DUT_China">
    </a>
    <button class="navbar-toggler" data-toggle="collapse" data-target="#nav1">
        <span class="navbar-toggler-icon"></span>
    </button>
    <div class="collapse navbar-collapse" id="nav1">
        <ul class="navbar-nav ml-auto">
            <li class="nav-item">
                <a class="nav-link" href="https://2020.igem.org/Team:DUT_China">
                    <img class="nav-logo" src="https://2020.igem.org/wiki/images/4/4a/T--DUT_China--n_home1.png" alt="">
                    <span>Home</span>
                </a>
            </li>
            <li class="nav-item dropdown">
                <a class="nav-link dropdown-toggle" data-toggle="dropdown">
                    <img class="nav-logo" src="https://2020.igem.org/wiki/images/3/3f/T--DUT_China--n_project1.png"
                         alt="">
                    <span>Project</span>
                </a>
                <div class="dropdown-menu">
                    <a class="dropdown-item" href="https://2020.igem.org/Team:DUT_China/Description">Description</a>
                    <a class="dropdown-item" href="https://2020.igem.org/Team:DUT_China/Engineering">Engineering
                        Success</a>
                    <a class="dropdown-item" href="https://2020.igem.org/Team:DUT_China/Protocols">Protocols</a>
                    <a class="dropdown-item" href="https://2020.igem.org/Team:DUT_China/Contribution">Contribution</a>
                </div>
            </li>
            <li class="nav-item dropdown">
                <a class="nav-link dropdown-toggle" data-toggle="dropdown">
                    <img class="nav-logo" src="https://2020.igem.org/wiki/images/4/4d/T--DUT_China--n_model.png" alt="">
                    <span>Model</span>
                </a>
                <div class="dropdown-menu">
                    <a class="dropdown-item" href="https://2020.igem.org/Team:DUT_China/Model">Electroporation</a>
                </div>
            </li>
            <li class="nav-item dropdown">
                <a class="nav-link dropdown-toggle" data-toggle="dropdown">
                    <img class="nav-logo" src="https://2020.igem.org/wiki/images/2/23/T--DUT_China--n_hp1.png" alt="">
                    <span>Human Practices</span>
                </a>
                <div class="dropdown-menu">
                    <a class="dropdown-item" href="https://2020.igem.org/Team:DUT_China/Human_Practices">Integrated
                        Human Practices</a>
                    <a class="dropdown-item" href="https://2020.igem.org/Team:DUT_China/Education">Education</a>
                </div>
            </li>
            <li class="nav-item dropdown">
                <a class="nav-link dropdown-toggle" data-toggle="dropdown">
                    <img class="nav-logo"
                         src="https://2020.igem.org/wiki/images/f/f6/T--DUT_China--n_implementation1.png" alt="">
                    <span>Implementation</span>
                </a>
                <div class="dropdown-menu">
                    <a class="dropdown-item" href="https://2020.igem.org/Team:DUT_China/Implementation">Business
                        Proposals</a>
                    <a class="dropdown-item" href="https://2020.igem.org/Team:DUT_China/Legislative">Legislative
                        Proposals</a>
                </div>
            </li>
            <li class="nav-item dropdown">
                <a class="nav-link dropdown-toggle" data-toggle="dropdown">
                    <img class="nav-logo" src="https://2020.igem.org/wiki/images/2/29/T--DUT_China--n_team1.png" alt="">
                    <span>Team</span>
                </a>
                <div class="dropdown-menu">
                    <a class="dropdown-item" href="https://2020.igem.org/Team:DUT_China/Team">Members</a>
                    <a class="dropdown-item" href="https://2020.igem.org/Team:DUT_China/Attributions">Attributions</a>
                    <a class="dropdown-item"
                       href="https://2020.igem.org/Team:DUT_China/Collaborations">Collaborations</a>
                </div>
            </li>
            <li class="nav-item">
                <a class="nav-link" href="https://2020.igem.org/Team:DUT_China/Safety">
                    <img class="nav-logo" src="https://2020.igem.org/wiki/images/0/04/T--DUT_China--n_safety1.png"
                         alt="">
                    <span>Safety</span>
                </a>
            </li>
            <li class="nav-item dropdown">
                <a class="nav-link dropdown-toggle" data-toggle="dropdown">
                    <img class="nav-logo" src="https://2020.igem.org/wiki/images/b/bb/T--DUT_China--n_awards1.png"
                         alt="">
                    <span>Awards</span>
                </a>
                <div class="dropdown-menu">
                    <a class="dropdown-item" href="https://2020.igem.org/Team:DUT_China/Human_Practices">Integrated
                        Human Practices</a>
                    <a class="dropdown-item" href="https://2020.igem.org/Team:DUT_China/Model">Model</a>
                    <a class="dropdown-item" href="https://2020.igem.org/Team:DUT_China/Education">Education</a>
                </div>
            </li>
        </ul>
    </div>
</nav>
<div id="top-scroll">
    <svg class="icon" style="width: 100px; height: 100px;vertical-align: middle;fill: #0073bb;overflow: hidden;"
         viewBox="0 0 1024 1024" version="1.1" xmlns="http://www.w3.org/2000/svg" p-id="2424">
        <path d="M4.621 514.463h98.188v327.301h98.188v-327.301h98.188v-65.449h-294.563zM528.209 154.51l-163.446 196.347 327.273-0.219zM331.907 841.765h327.286v-392.75h-327.286v392.75zM430.095 514.463h130.912v261.823h-130.912v-261.823zM691.933 449.015v392.75h98.188v-163.651h229.1v-229.1h-327.286zM921.033 612.652h-130.912v-98.188h130.912v98.188z"
              p-id="2425"></path>
    </svg>
</div>
<div id="sidebar">
    <table>
        <tr>
            <td>
                <canvas width="40" height="40" class="canvas"></canvas>
            </td>
            <td class="side-item">1 Our Process</td>
        </tr>
        <tr>
            <td>
                <canvas width="40" height="40" class="canvas"></canvas>
            </td>
            <td class="side-item">2 Obtain whole T4 genome via PCR</td>
        </tr>
        <tr>
            <td>
                <canvas width="40" height="40" class="canvas"></canvas>
            </td>
            <td class="side-item">3 Preparation of plasmids and vectors</td>
        </tr>
        <tr>
            <td>
                <canvas width="40" height="40" class="canvas"></canvas>
            </td>
            <td class="side-item">4 Switching the Mating type of <i>S. cerevisiae</i></td>
        </tr>
        <tr>
            <td>
                <canvas width="40" height="40" class="canvas"></canvas>
            </td>
            <td class="side-item">5 Collection and assembly of T4 genome in <i>S. cerevisiae</i></td>
        </tr>
        <tr>
            <td>
                <canvas width="40" height="40" class="canvas"></canvas>
            </td>
            <td class="side-item">6 Phage T4 rebooting</td>
        </tr>
        <tr>
            <td>
                <canvas width="40" height="40" class="canvas"></canvas>
            </td>
            <td class="side-item">7 References</td>
        </tr>
    </table>
</div>
<div id="title" class="col-12">
    <div>Engineering Success</div>
</div>
<section id="main-text">
    <object classid="clsid:CA8A9780-280D-11CF-A24D-444553540000" width="100%" height="500" border="0">
        <param name="SRC" value="https://2020.igem.org/wiki/images/7/72/T--DUT_China--n_eng_0.pdf">
        <embed fullscreen="yes" src="https://2020.igem.org/wiki/images/7/72/T--DUT_China--n_eng_0.pdf" width="100%"
               height="800" align="middle">
    </object>
    <p class="caption">Spiral Model of Software Development by Barry Boehm</p>
    <article class="article">
        <h1>1 Our Process</h1>
        <h2>(January-February 2020)</h2>
        <p>(1) Determine goals, choices and constraints:</p>
        <p>
            We hope to design a universal method that allows the main body of the phage to be adapted to assemble
            different tail fibers, thereby changing its host range, so as to achieve quantitative control of the
            intestinal flora, detection of food-borne pathogens, and treatment of intracellular parasitic infections.
        </p>
        <p>(2) Risk analysis:</p>
        <p>
            How to achieve what is expected in the design: the assembly process of the phage body and the different tail
            fibers. The process needs to meet the following requirements: easy to operate and capable of producing a
            large number of hybrid phages with heterologous body and tail fibers.
        </p>
        <p>(3) Build a prototype:</p>
        <p>
            We hope to design a universal platform to assemble different tail fibers with a shared main body of another
            phage. For example, through our platform, we can hybridize the main body of phage A with tail fibers of
            phage B, or phage C, etc. The host range is thereby modified. In order to produce a large number of hybrid
            phages with heterologous main body and tail fibers, we designed a method for assembling the body-tail fiber
            hybrid phages <i>in vitro</i>. The method was designed to use artificial main body of T4 phage and tail
            fibers of other phages. Tail fiber ligase was proved to simulate the intracellular assembly process of
            phage. In our expectation, we will invent a detection instrument which contains several tail fibers.
            Whenever mixing the main body of T4 phage, the tail fibers and tail fiber ligase in this instrument, the
            hybrid phages could be produced. When spraying those hybrid phages onto the sample, the pathogenic bacteria
            would be infected.
        </p>
        <p>
            Then we started to design the method to obtain main body of T4 phage, tail fibers and tail fiber ligase.
            Tail fibers and the ligase could be produced through conventional genetic engineering. According to one of
            our senior sisters, the tail fibers could be separated from the phage body by rude shock (the data have not
            been published), so that the main bodies of T4 phages could be harvested by this way.
        </p>
        <object classid="clsid:CA8A9780-280D-11CF-A24D-444553540000" width="100%" height="500" border="0">
            <param name="SRC" value="https://2020.igem.org/wiki/images/6/68/T--DUT_China--n_eng_f1.pdf">
            <embed fullscreen="yes" src="https://2020.igem.org/wiki/images/6/68/T--DUT_China--n_eng_f1.pdf" width="100%"
                   height="800" align="middle">
        </object>
        <p class="caption">
            <b>Figure 1.</b> Scenario diagram of the application of our first project.<br>
            We package main body of phage T4, tail filament ligases and several tail filaments as capsules and put them
            into a toolkit. The main instrument of the detection of bacteria is a bacterial screening gun, in whose
            handle the capsules are slipped and the phages with various tail filaments would be generated. When spraying
            the phages onto surface of the samples, the existing bacteria would be infected and some phenomenon would be
            observed.
        </p>
        <p>(4) Test and verify the prototype:</p>
        <p>
            Through further investigation, we learned that the assembly process of phage is not just a simple enzymatic
            reaction. It needs the support of the intracellular environment and various complex metabolic pathways, so
            it is difficult to directly assemble <i>in vitro</i>; in addition, the phage body and tail fibers used as
            raw materials are difficult to store stably for a long time, and it is also difficult to activate and
            assemble them immediately.
        </p>
        <p>(5) The next stage of the plan:</p>
        <p>
            We decided to change the phage assembly process from <i>in vitro</i> to intracellular, and no longer use
            phage main body and tail fibers as hybrid materials, but directly carry out the allogeneic combination of
            gene sequences, and then use the natural way of phage replication to obtain hybrid phage.
        </p>
        <h2>(February-March 2020)</h2>
        <p>(6) Determine goals, choices and constraints:</p>
        <p>
            We hope to design a production process that can replace specific genes or gene clusters of bacteriophages
            from phages with allogenic bacteriophage genes for intracellular expression to produce a hybrid phage with
            new traits and multifunctional properties.
        </p>
        <p>(7) Risk analysis:</p>
        <p>
            Adding exogenous genes to phages can lead to changes in their genomes, causing biosafety problems. We need
            to devise a scheme in which artificial hybrid phages can survive and function, but cannot reproduce in the
            natural environment.
        </p>
        <p>(8) Prototyping:</p>
        <p>
            We have designed a standard hybrid phage production process, taking the modification of the T4 phage tail
            fiber as an example. This process requires first removing the tail fiber gene from the T4 phage genome, and
            then using a plasmid vector to carry the heterologous tail fiber gene. The defective T4 genome and the
            plasmid vector are co-introduced into the same <i>E. coli</i>, and the expression and assembly are carried
            out in the cells respectively to produce hybrid phage with defective genome. The phage is the final product.
            It has a heterologous tail, and a defect in the genome preventing it from reproducing in nature.
        </p>
        <p><a href="#f2">(Refer to <b>Figure 2</b>)</a></p>
        <p>(9) Test and verify the prototype:</p>
        <p>
            Through literature research, we found some problems: First, the specificity of the phage to the host is
            related to many factors. In addition to the receptor binding protein on the tail fiber, the structure of the
            substrate, the short tail fiber, and the lyase will all affect The choice of phage on the host. Secondly,
            the capsid proteins of phages may be packaged into exogenous plasmids during genomic packaging, which means
            that the gene defects of hybrid phages may be remedied, causing potential biosafety risks.
        </p>
        <p>(10) Next stage of the plan:</p>
        <p>
            We decided to change our thinking and change our goal to reactivate the phage inside the cell using the
            genome imported into the cell. This method can not only serve as the technical basis for phage genome
            replacement, but also provide a possible way for phage genetic modification and foreign gene addition.
        </p>
        <h2>(March-August 2020)</h2>
        <p>(11) Determine goals, choices and constraints:</p>
        <p>
            We hope to design a method that would divide a phage genome into multiple segments and build a phage library
            based on those segments. When modifying genes, we can manipulate specific segments to reduce the impact on
            other genes. Finally, we only need to connect and express these segments in order to produce modified
            phages.
        </p>
        <p>(12) Risk analysis:</p>
        <p>
            The combination and expression of the segments need to be carried out in the cell, so we need a technology
            to ensure that the plasmid transporting these segments can be introduced into the same cell simultaneously
            and equally, and can be stably present and expressed in the cell.
        </p>
        <p>(13) Prototyping:</p>
        <p>
            We designed a set of intracellular rebooting process of phage genome. Take the rebooting of T4 phage as an
            example. First, we performed segmental PCR processing on the genome of T4 phage, and obtained 26 small
            segments with a sequence length of 7-8kbp. According to their order in the genome, they were divided into
            four groups with similar numbers. Each group was connected by Overlapping Extension PCR to produce four
            large segments of about 42 kbp in length. Next, we use the artificial yeast chromosomes (YAC) with different
            antibiotic resistance genes as the vectors to transform four large fragments into the same <i>E. coli</i>.
            Then YACs were screened with four antibiotics to obtain the <i>E. coli</i> with all four YACs. After that,
            we introduced the CRISPR system into the <i>E. coli</i> to achieve linearization and recombination of the
            four segments. Finally, the complete T4 phage genome is expressed in <i>E. coli</i>, and the rebooting of
            the phage can be achieved.
        </p>
        <p>
            Besides using four antibiotics to screen <i>E. coli</i> transformant, we found another way to achieve this
            goal. Kanamycin resistance of bacteria comes from the interaction of T7rnap gene and neo gene. T7rnap gene
            can express T7 RNA polymerase, which in turn can activate neo gene expression. This process provides the
            bacteria with kanamycin resistance, and these bacteria can survive on the medium containing kanamycin.
        </p>
        <p>
            So, we plan to divide T7rnap gene into two segments: core fragment and σ fragment. They are regulated by the
            same promoter, so that they have similar expression levels and combine to produce active T7 RNA polymerase.
            In the same way, neo gene can be divided into two parts: neo-α and neo-β. Only when the two fragments exist
            at the same time, can neo gene function fully.
        </p>
        <p>
            Therefore, we can use four genes to mediate kanamycin resistance and construct a four-plasmid-coexistence
            system. Only when the four plasmids exist at the same time and have the appropriate expression level, can
            the bacteria grow on the medium containing kanamycin to achieve the purpose of screening.
        </p>
        <p>(14) Test and verify the prototype:</p>
        <p>
            During the evaluation of this process, we noticed two problems: Firstly, technically, the process relies on
            a complex multi-plasmid coexistence system, which requires four plasmids to enter the cell at the same time
            and have similar copies in the cell. This is difficult to control; secondly, we learned through
            investigation that it is achievable to introduce a huge plasmid (about 170kbp, the size of the genome of the
            T4 phage) into the cell. Therefore, we can avoid the use of multi-plasmid coexistence system by connecting
            fragments in advance, so as to simplify the process.
        </p>
        <p>(15) Next stage of the plan:</p>
        <p>
            We believe that it is necessary to use large plasmid transduction instead of multi-plasmid transduction,
            namely firstly connect the four plasmids constructed outside of <i>E. coli</i> with the CRISPR system to
            obtain a YAC plasmid carrying the complete phage genome, and then introduce it <i>E. coli</i>.
        </p>
        <h2>(August-October 2020)</h2>
        <p>(16) Determine goals, choices and constraints:</p>
        <p>
            We hope to improve the original process, abandon the complex multi-plasmid system, and use a more convenient
            method to completely introduce the phage genome into the host cell. This method needs to simplify the
            operator's quantitative control of each parameter, and make the detection of each step more accurate,
            macroscopic and visualized as much as possible.
        </p>
        <p>(17) Risk analysis:</p>
        <p>
            The ligation of fragments requires the help of the CRISPR system, which has a higher success rate in the
            intracellular environment. Therefore, we need to use new methods to integrate the four plasmids into the
            same cell and avoid the use of conventional transformation methods.
        </p>
        <p>(18) Prototyping:</p>
        <p>
            Based on the phage genome, the use of segmented PCR to construct four plasmids is the same as before, and
            the constructed plasmids will be introduced into yeast cells respectively. Then, using the mating induced
            fusion phenomenon between the different genders of yeast, the two yeasts were first fused to form diploid
            yeasts to achieve the integration of the two plasmids, and then the CRISPR system was introduced into them
            to connect to form two fragments of about 84kbp. Next, the fusion between the two diploid yeasts is carried
            out, and the complete phage genome is formed by the same method. Finally, we only need to extract this
            complete large plasmid and introduce it into <i>E. coli</i> to restart the phage.
        </p>
        <p><a href="#f4">(Refer to <b>Figure 4</b>)</a></p>
    </article>
    <article class="article">
        <h1>2 Obtain whole T4 genome via PCR</h1>
        <p>
            To decrease the potential difficulties in direct manipulation of the huge coliphage T4 genome, the 168.9 kbp
            T4 genome is divided into four parts (~40 kbp each). Every part is then divided into six or seven fragments
            (7~8 kbp each), which can be amplified conveniently via PCR and using the natural T4 genome as the template.
        </p>
        <p>
            The primers of two contiguous fragments are designed with overlap sequences to mediate the homologous
            combination between each couple of contiguous fragments. Each single PCR product is about 7~8 kbp. Six or
            seven contiguous PCR products containing homologous arms, along with the corresponding linearized yeast
            vector, are co-introduced into <i>Saccharomyces cerevisiae</i> via electroporation. The primers used for
            amplification of phage genomic fragments are listed in <b>Table 1</b>.
        </p>
        <p class="caption">
            <b>Table 1.</b> All primers used in PCR amplify wide-type T4 genome. 1~4 refer to the four parts of the T4
            genome. Fragment A~F, G~L, M~Q, R~W, are continuous gene fragments of part 1, 2, 3, 4, respectively. The
            temperatures of every fragments are calculate using our mathematic model.
        </p>
        <img src="https://2020.igem.org/wiki/images/3/3e/T--DUT_China--n_eng_t1.png" alt="" class="col-12">
        <p>
            Though it is technically feasible to amplify larger phage genomic fragments, we still chose to amplify the
            7~8 kbp fragments because of the accuracy of PCR amplification. On the one hand, the efficiency of
            transformation-associated recombination (TAR) in yeast is limited by the number of linearized DNA fragments.
            It was regarded that if there are too many fragments used for TAR, the correct transformants with all
            assembled fragments would be precious few. On the other hand, if the gene fragments are too small in size,
            the efficiency of homologous combination could be decrease. As a result, we choose 7~8 kbp as the size of
            every piece of PCR products.
        </p>
        <p>
            Agarose gel electrophoresis is applied to evaluate the results of PCRs. The unexpected results could be (I)
            no specific DNA bands; (II) bands with unexpected relative molecular weight; (III) poor yield of PCR
            products, which are possibly due to the mistakes in designing the upstream and downstream primers, and the
            inappropriate temperatures of PCRs. We would go over the primers of the unexpected products, while resetting
            the PCR parameters. If the above three unexpected results always occur in a specific PCR, the WT phage T4
            genome should be sequenced, because the template could be mutated or changed. If (IV) all the DNA bands were
            dim or even no products were obtained, the reason might be the inactivation of DNA polymerase and the T4
            genome, which would be then rebought and reextracted.
        </p>
    </article>
    <article class="article">
        <h1>3 Preparation of plasmids and vectors</h1>
        <p>
            Firstly, we design two plasmids for the switch of mating type of <i>Saccharomyces cerevisiae</i>. The
            pTrp-gRNAα (<b>Figure 2a</b>) and pTrp-gRNAa (<b>Figure 2b</b>) express guide RNA targeting Yα and Ya
            sequences, respectively. The Ya and Yα sequences used as the repairing fragments are amplified via PCR.
            Briefly, when co-transforming the <i>MATα</i> yeast cells with pTrp-gRNAα and Ya fragment, and whenever the
            Cas9 protein is expressed in yeast, the <i>MATα</i> cells will be converted to <i>MATa</i> cells.
        </p>
        <div class="d-flex flex-wrap justify-content-center" id="f2">
            <img src="https://2020.igem.org/wiki/images/b/bb/T--DUT_China--n_eng_f2a.jpg" alt=""
                 class="col-sm-10 col-md-5">
            <img src="https://2020.igem.org/wiki/images/4/4e/T--DUT_China--n_eng_f2b.jpg" alt=""
                 class="col-sm-10 col-md-5">
        </div>
        <p class="caption">
            <b>Figure 2a.</b> Gene map of pTrp-gRNAα. The left map is pTrp-gRNAα, in which the gRNA targets sequence Yα.
        </p>
        <p class="caption">
            <b>Figure 2b.</b> Gene map of pTrp-gRNAa. The right map is pTrp-gRNAa, in which the gRNA targets sequence
            Ya.
        </p>
        <p>
            The Cas9 expression plasmid pMet-Cas9 (<b>Figure 3a</b>) and gRNA expression plasmid pTrp-gRNA-S1S2 (<b>Figure
            3b</b>) are obtained from Prof. Zhongjun Qin at the Center for Excellence in Molecular Plant Science,
            Chinese Academy of Sciences. They are electroporated into <i>S. cerevisiae</i> and amplified apart. The
            harvested yeasts are broken and the plasmids are separated. To increase the universality and reduce the
            consumption of human and material resources, the Cas9 expression plasmid are expected existing in yeast
            cells stably. The gRNA is controlled by lac promotor, which is expected to trigger the dsDNA scissors at the
            specific time we want. Our plasmid that express Cas9 protein and gRNA was designed as <b>Figure 1 A</b>.
        </p>
        <div class="d-flex flex-wrap justify-content-center">
            <img src="https://2020.igem.org/wiki/images/b/b8/T--DUT_China--n_eng_f3a.png" alt=""
                 class="col-sm-10 col-md-5">
            <img src="https://2020.igem.org/wiki/images/1/14/T--DUT_China--n_eng_f3b.jpg" alt=""
                 class="col-sm-10 col-md-5">
        </div>
        <p class="caption">
            <b>Figure 3a.</b> Gene map of pMet-Cas9. This plasmid is expected to exist stably in yeast cells.
        </p>
        <p class="caption">
            <b>Figure 3b.</b> Gene map of pTrp-gRNA-S1S2. This plasmid mediates the separation of the genome fragments
            and linearized vectors.
        </p>
        <p>
            We use linearized pRS42X series vectors with different marker genes, pRS422 (<i>ADE2</i>), pRS423
            (<i>HIS3</i>), pRS426 (<i>URA3</i>) and pRS427 (<i>LYS2</i>). Given that our linearized plasmids are
            amplified via PCR, to eliminate the contamination of cyclical plasmids, we used SmaI digested vectors as
            templates for PCR amplification. We will use Agarose gel electrophoresis to evaluate the results of PCR. The
            potential unexpected results and the troubleshooting processes are similar to those mentioned above. (<b>Figure
            4</b>)
        </p>
        <p id="f4"></p>
        <img src="https://2020.igem.org/wiki/images/e/ee/T--DUT_China--n_eng_f4a.jpg" alt="" class="col-12">
        <img src="https://2020.igem.org/wiki/images/e/e7/T--DUT_China--n_eng_f4b.jpg" alt="" class="col-12">
        <img src="https://2020.igem.org/wiki/images/6/61/T--DUT_China--n_eng_f4c.jpg" alt="" class="col-12">
        <img src="https://2020.igem.org/wiki/images/2/2b/T--DUT_China--n_eng_f4d.jpg" alt="" class="col-12">
        <img src="https://2020.igem.org/wiki/images/5/59/T--DUT_China--n_eng_f4e.jpg" alt="" class="col-12">
        <img src="https://2020.igem.org/wiki/images/3/36/T--DUT_China--n_eng_f4f.jpg" alt="" class="col-12">
        <img src="https://2020.igem.org/wiki/images/b/b8/T--DUT_China--n_eng_f4g.jpg" alt="" class="col-12">
        <p class="caption">
            <b>Figure 4.</b> The seven linearized vectors.<br>
            The grey parts on both the ends is the homologous arms which homologous with sequences of the T4 genomic
            fragments. The selection yeast marker genes are <i>HIS3</i>, <i>URA3</i>, and <i>ADE2</i>, respectively.
        </p>
    </article>
    <article class="article">
        <h1>4 Switching the Mating type of <i>S. cerevisiae</i></h1>
        <p>
            <b>Figure 5</b> shows the schematic diagram for switching the mating type of yeast. The yeast mating type
            determined sequence, Yα and Ya sequence, are PCR amplified, and the templates are de novo synthesized. The
            Cas9 expression plasmid pMet-Cas9, pTrp-gRNAα plasmid for the expression of gRNA targeting the <i>MATα</i>
            locus, and the Ya fragment homologous with Yα sequence, are co-electroporated into <i>S. cerevisiae</i>
            VL6-48 (<i>MATα</i>) to convert it from <i>MATα</i> to <i>MATa</i>. The transformants are selected on
            SC-MET-TRP (omitting methionine and tryptophan) medium for 2 days. The Cas9 expression plasmid contains
            another gRNA targeting the plasmid pTrp-gRNAα, which is controlled by <i>GAL1</i> promoter, so that the gRNA
            expression plasmid pTrp-gRNAα could be linearized and degraded in the yeast cells on SC-MET medium. As a
            result, the auxotrophic marker gene <i>TRP1</i> is lost by the yeast cells, and the gene could be reused to
            select other introduced plasmids. The Cas9 expression plasmid is expected to exist in the yeast stably, for
            Cas9 protein would be employed to generate other double-strain breaks in further steps.
        </p>
        <img src="https://2020.igem.org/wiki/images/7/7a/T--DUT_China--n_eng_f5.png" alt="" class="col-12">
        <p class="caption">
            <b>Figure 5.</b> Diagrammatic sketch of switching the mating type of <i>S. cerevisiae</i>.<br>
            The green linearized sequence Yα and the blue linearized sequence Ya determine the mating type of <i>S.
            cerevisiae</i> as <i>MATα</i> and <i>MATa</i>, respectively. The red arrows represent the double-strain
            breaks caused by CRISPR/Cas9. The break at Yα would result in homologous recombination between Yα and Ya, so
            that the mating type of <i>S. cerevisiae</i> could be switched. The break at gRNA expression plasmids can
            lead to degradation.
        </p>
        <p>
            Simultaneously, we build the <i>MATα</i> yeast cells harboring the same Cas9 expression plasmid by
            electroporating. The transformants are selected on SC-TRP (omitting tryptophan) medium for 2 days, too.
        </p>
        <p>
            The switched mating type is tested by crossing the transformants with tester strains 17α and 14a, both
            carrying the <i>HIS3</i> auxotrophic marker. Lawns of tester strains are prepared by spreading 500 ml
            overnight YPD culture of tester strains on YPD agar plates and incubating at 30 degree Celsius for 2 days.
            The lawns and transformant colony plates are replica plated onto a fresh YPD plate at the same time. A
            second replica plating onto synthetic dextrose medium is performed after incubating at 30 degree Celsius
            overnight, and the plates are incubated at 30 degree Celsius to reflect mating response and scored at the
            next day. Haploid strains and <i>MATα/α</i> or <i>MATa/a</i> diploid strains should form a patch on just one
            of the two tester strains if the tester strain is of the opposite mating-type. <i>MATa/a</i> strains are
            nonmaters and will not form a prototrophic patch with either tester strain.
        </p>
        <p>
            The unexpected results could be low efficiency of mating-type switching, limited by the efficiency of
            co-electroporation and homologous recombination. To our knowledge, the strategies to increase the harvest of
            correct sex-converted yeast cells are culturing and manipulating more yeast cells and change the reagents if
            necessary. Of course, if the harvest cells are still few, we will redesign our gRNA expression plasmids,
            because we regard the low efficiency of generating double strand breaks as the main reason. Also, we will
            add more Ya fragment into the electroporation cuvette to increase the content of the Ya sequence in
            transformants. The concentration of other pre-introduced plasmids will be reset to find out the most
            suitable electroporation conditions.
        </p>
    </article>
    <article class="article">
        <h1>5 Collection and assembly of T4 genome in <i>S. cerevisiae</i></h1>
        <p><b>Figure 6</b> shows the schematic diagram of the following manipulation in yeasts.</p>
        <img src="https://2020.igem.org/wiki/images/a/a4/T--DUT_China--n_eng_f6.png" alt="" class="col-12">
        <p class="caption">
            <b>Figure 6.</b> Assembly of the phage T4 genome in yeast.<br>
            The haploid yeasts are arranged at the top of this diagram. They are transformed with group 1 to 4,
            respectively. The red arrows represent double strand breaks caused by CRISPR/Cas9 system. The vermeil,
            yellow, violet, clear green and sapphire squares represent homologous sequences mediated assembly, and the
            blue strips represent PCR products of T4 genome. The gene fragments, plasmids and vectors emerging in the
            yeast cells are all the nucleic acid elements associated with our project in corresponding cells.
        </p>
        <p>
            As mentioned above, we obtain all plasmids and vectors used in our project, and the T4 genome is also
            obtained as 26 7~9 kbp fragments. The fragments and linearized YAC backbones are PCR amplified separately.
            Then every group, including a linearized YAC backbone and corresponding fragments of T4 genome, are
            co-electroporated into yeast cells and circulated to the YAC-phage plasmid. Specifically, group 1 and group
            3 are introduced into <i>MATα</i> yeast cells with the linearized YAC backbones with auxotrophic marker <i>HIS3</i>;
            while group 2 and 4 are introduced into <i>MATa</i> strains with the backbones with auxotrophic marker <i>URA3</i>.
            The conditions for co-electroporation are described previously (Ando et al., 2015). Then we select the four
            transformants on corresponding minimal medium, SC-MET-HIS (omitting methionine and histidine) and SC-MET-URA
            (omitting methionine and uracil), respectively. After two days, the single colonies are then picked out and
            inoculated on another fresh synthetic medium, simultaneously colony PCR is operated to check whether the
            linearized fragments are assembled successfully via homologous recombination. Colony PCRs are carried out
            for all single colonies until the discovery of transformants harbored the correctly assembled cyclical
            YAC-phage plasmid. The primers utilized in colony PCR are listed in <b>Table 1</b>. The four obtained yeast
            strains are named <i>yMATa-1His-Cas9Met</i>, <i>yMATα-2Ura-Cas9Met</i>, <i>yMATa-3His-Cas9Met</i>, and <i>yMATα-4Ura-Cas9Met</i>.
        </p>
        <p>
            We use mating-mediated fusion and Cas9-facilitated homologous recombination assembly (Zhou et al., 2016) for
            the construction of a huge plasmid containing the whole phage T4 genome. First we cross <i>yMATa-1His-Cas9Met</i>
            and <i>yMATα-2Ura-Cas9Met</i> mediated by yeast mating. The diploid strains are selected on the synthetic
            medium SC-MET-HIS-URA (omitting methionine, histidine and uracil) for 2 days. The resulting diploids were
            then transformed with gRNA expression plasmid pTrp-gRNA-S1S2, and another linearized YAC backbone with
            auxotrophic marker <i>ADE2</i>, followed by culturing and selecting the transformants on SC-MET-TRP-ADE
            (omitting methionine, tryptophan and adenine) for 2 days. The harvest single colonies are picked out and
            colony PCR is carried out once again to check whether the two huge fragments have been released from the
            corresponding plasmids and assembled with the later-introduced linearized YAC vector. The selected strains
            are then inoculated on minimal medium SC-MET-ADE (omitting methionine and adenine). Simultaneously, the gRNA
            in pMet-Cas9 is induced by galactose eliminate plasmid gTrp-gRNA-S1S2, so that the diploid yeast only
            contains YAC-phage plasmid pADE-phage12 and Cas9 expression plasmid pMET-Cas9. The harvest diploid yeast is
            named <i>yMATa/MATα-12Ade-Cas9Met</i>, which would be then mating switched to <i>MATα/MATα</i> as described
            above.
        </p>
        <p class="caption">
            <b>Table 2.</b> Primers of colony PCR.<br>
            Colony PCR would be used to detect whether the fragments have been assembled successfully. Fragment II and
            III, fragment IV and I should be amplified and detected after the fusing of <i>yMATα/MATα-12Ade-Cas9Met</i>
            and <i>yMATa/MATa-34Lys-Cas9Met</i>, and the other fragments should be amplified and detected after the
            first fusions.
        </p>
        <img src="https://2020.igem.org/wiki/images/3/36/T--DUT_China--n_eng_t2.png" alt="" class="col-12">
        <p>
            Simultaneously, <i>yMATa-3His-Cas9Met</i> and <i>yMATα-4Ura-Cas9Met</i> are fused and manipulated through a
            similar strategy. Besides, the later-introduced linearized YAC vector is marked with auxotrophic marker <i>LYS2</i>.
            As a result, after using the corresponding medium to culture the yeast cells, the harvest diploids should be
            <i>yMATa/MATα-34Lys-Cas9Met</i>, which would be then converted to <i>MATa/MATa</i>.
        </p>
        <p>
            The same strategy was utilized to convert <i>MATa/MATα</i> to <i>MATα/MATα</i> or <i>MATa/MATa</i> and lost
            the gRNA expression plasmids (Xie et al., 2018). The obtained yeast strains are named <i>yMATα/MATα-12Ade-Cas9Met</i>
            and <i>yMATa/MATa-34Lys-Cas9Met</i>, respectively. Then we cross <i>yMATα/MATα-12Ade-Cas9Met</i> and <i>yMATa/MATa-34Lys-Cas9Met</i>
            through the same methodology as described above.
        </p>
        <p>
            To be on the safe side, we keep yeasts which are able to survive on the minimal medium and contain the
            expected plasmids proliferation on the corresponding plates and transplanting regularly. These operations
            support the possibility to check out where the mistakes occur and prevent making a fresh start when meeting
            some other unexpected results. The proliferated yeast cells could amplify more needed plasmids so that the
            costs of operation mistakes could be decreased.
        </p>
        <p>
            The unexpected results could be the off-targeting phenomenon of the CRISPR/Cas9 editing system. We should
            ensure the reconstructed T4 genome would not be destroyed by Cas9 protein, so that the potential off-target
            phenomenon will be checked out and redesign the gRNA if necessary. Restriction enzyme mapping and agarose
            gel electrophoresis are used to detect if the extracted YAC-T4 plasmid is complete or not.
        </p>
        <p>
            The reconstructed T4 genome has to escape from the designed CRISPR/Cas9. A restriction endonuclease map
            would be carried out with RsrII, BssHII, FspAI, and SgrDI. The four restriction enzymes are the unique
            cutters of both the T4 genome and the linearized YAC vector. If the off-target or double-strain breaks
            occur, there would be more than 4 strips when the AGE is carried out. As a result, the CRISPR/Cas9 system
            should be redesigned.
        </p>
        <p>
            The gel recovery would be left out, for the restriction enzymes would be cut the giant genome into several
            pieces. On the one hand, gel recovery might cause huge wastage and break of giant-sized plasmids. On the
            other hand, the concertation of YAC-T4 plasmids could be decrease, which might defeat the followed
            electroporation.
        </p>
        <p class="caption">
            <b>Table 3.</b> The yeast strains used in our project.<br>
            The parent strain is <i>S. cerevisiae</i> VL6-48 (MATα, trp1-Δ1, ura3-Δ1, ade2-101, his3-Δ200, lys2, met14).
            All the other strains are obtained from our project.
        </p>
        <img src="https://2020.igem.org/wiki/images/5/5e/T--DUT_China--n_eng_t3.png" alt="" class="col-12">
        <p class="caption">
            <b>Table 4.</b> Plasmids used in our project.
        </p>
        <img src="https://2020.igem.org/wiki/images/5/50/T--DUT_China--n_eng_t4.png" alt="" class="col-12">
    </article>
    <article class="article">
        <h1>6 Phage T4 rebooting</h1>
        <p>
            After assembling the whole T4 genome in <i>S. cerevisiae</i>, we beak the yeast cells and obtain the YAC-T4
            plasmid. The huge plasmid is then purified via agarose gel electrophoresis. We expect with the bands of the
            correct size (about 175 kbp, including a 168.9 kbp genome and a 6.3 kbp vector). There might be several
            stripes of diverse size for the yeast cells also contain other plasmids, so that cutting gel was necessary.
        </p>
        <p>
            The obtained YAC-T4 plasmids are then introduced into competent <i>E. coli</i> cells via electroporation. To
            avoid the potential results, such as low transformation efficiency or even no transformants generated,
            preliminary experiments are designed to determine parameters roughly and parallel experiments are operated
            to obtain enough transformants. Mathematical modeling is also built to predict the potential range of the
            parameters. For more detailed description of our model, please click <a
                href="https://2020.igem.org/Team:DUT_China/Model">here</a>.
        </p>
        <p>
            The <i>E. coli</i> transformants are incubated in LB liquid medium for 3~4 hours at 37 degrees Celsius,
            followed by adding drops of chloroform into the supernatants to assist lyse. The centrifugal supernatants
            are then mixed with an overnight culture of <i>E. coli</i>, and the phages are amplified overnight at 37
            degrees Celsius.
        </p>
    </article>
    <article class="article">
        <h1>7 References</h1>
        <p>
            [1]Ando, H., Lemire, S., Pires, D. P., & Lu, T. K. (2015). Engineering Modular Viral Scaffolds for Targeted
            Bacterial Population Editing. <i>Cell Systems, 1</i>(3), 187–196.
        </p>
        <p>
            [2]Xie, Z.-X., Mitchell, L. A., Liu, H.-M., Li, B.-Z., Liu, D., Agmon, N., Wu, Y., Li, X., Zhou, X., Li, B.,
            Xiao, W.-H., Ding, M.-Z., Wang, Y., Yuan, Y.-J., & Boeke, J. D. (2018). Rapid and Efficient
            CRISPR/Cas9-Based Mating-Type Switching of <i>Saccharomyces cerevisiae. G3&amp;#58</i>; <i>Genes|Genomes|Genetics,
            8</i>(1), 173–183.
        </p>
        <p>
            [3]Zhou, J., Wu, R., Xue, X., & Qin, Z. (2016). CasHRA (Cas9-facilitated Homologous Recombination Assembly)
            method of constructing megabase-sized DNA. <i>Nucleic Acids Research, 44</i>(14), e124–e124.
        </p>
    </article>
    <p class="text-right">
        <a href="https://2020.igem.org/Team:DUT_China/Protocols">Protocols&#10148;</a>
    </p>
</section>
<footer id="footer">
    <img class="flower flower1" src="https://2020.igem.org/wiki/images/0/05/T--DUT_China--flower1.png" alt="">
    <img class="flower flower2" src="https://2020.igem.org/wiki/images/0/0b/T--DUT_China--flower2.png" alt="">
    <p class="copyright">© 2020 Dalian University of Technology iGEM Team</p>
    <div class="row">
        <div class="col-lg-4 footer-left">
            <h2><span>DUT</span> iGEM Players</h2>
            <img src="https://2020.igem.org/wiki/images/d/d7/T--DUT_China--school_badge1.svg" alt="School Logo">
            <p class="plain-text text-left">
                <b>Address</b>: No.2 Linggong Road, Ganjingzi District, Dalian City, Liaoning Province, P.R.C.,
                116024<br>
                <b>Official Website</b>: <a href="http://en.dlut.edu.cn/" target="_blank">http://en.dlut.edu.cn/</a><br>
                <b>Team Email</b>: <a href="mailto:igem_dlut@163.com" title="Click to contact us">igem_dlut@163.com</a>
            </p>
        </div>

        <div class="col-lg-4 footer-center">
            <div>
                <h4>About Dalian University of Technology</h4>
                <p class="plain-text school-intro text-left">
                    Dalian University of Technology (DUT) is a national key university administrated directly under the
                    Ministry of Education of China, which is also sponsored by Project 211 and Project 985. DUT keeps
                    fostering elites, promoting science and technology, inheriting excellent cultures and guiding the
                    social climate as its mission, adheres to DUT spirit of “Unity and Progress, Truth and Innovation”,
                    dedicates itself to the creation, discovery, impart, conservation and application of knowledge, and
                    endeavors to undertake social responsibilities to serve the country and the world.
                </p>
            </div>
        </div>

        <div class="col-lg-4 footer-right">
            <div>
                <h4>Follow us</h4>
                <p class="plain-text">
                    <a target="_blank" href="https://mp.weixin.qq.com/s/gDbIeINj-mwxmwDkOSyFQg">
                        WeChat Official Account
                    </a>
                    <br>
                    <a target="_blank" href="https://space.bilibili.com/494554805/">
                        Bilibili Account
                    </a>
                </p>
            </div>
            <br>
            <div>
                <h4>Sponsors</h4>
                <img src="https://2020.igem.org/wiki/images/2/29/T--DUT_China--sponsor_snapgene.png" alt="SnapGene">
                <br>
                <img src="https://2020.igem.org/wiki/images/e/e3/T--DUT_China--biology1.svg" alt="School of Bioengineering">
            </div>
        </div>
    </div>
</footer>
<style>
    .side-item {
        font-size: 12px;
    }
</style>
<script>
    //About the sidebar & top-scroll--linked to sidebar.js & general.js
    window.onload = function () {
        let navHeight = document.getElementById('nav-bar').offsetHeight;
        for (let i = 0; i < itemArray.length; i++) {
            itemArray[i].addEventListener('click', function () {
                scroll({
                    top: articleArray[i].offsetTop - navHeight + 5,
                    //To prevent the overshadow of the navigation bar
                    left: 0,
                    behavior: 'smooth'
                });
            });
            //Click the items to scroll
            pageView(canvasArray[i], itemArray[i], articleArray[i]);//Update
        }
        setTopScroll();
    };

    window.onresize = window.onscroll = function () {
        for (let i = 0; i < canvasArray.length; i++) {
            pageView(canvasArray[i], itemArray[i], articleArray[i]);//Update
        }
        let textT = document.getElementById("main-text").offsetTop;
        let currentH = document.documentElement.scrollTop || document.body.scrollTop;
        let sidebar = document.getElementById("sidebar");
        let topScroll = document.getElementById("top-scroll");
        if (currentH > textT - 130) {
            sidebar.setAttribute("class", "sidebar-active");
            topScroll.setAttribute("class", "top-scroll-active");
        } else {
            sidebar.setAttribute("class", "");
            topScroll.setAttribute("class", "");
        }
    };
</script>
</body>
</html>